### DISCUSSION TITLE
Proposal: Keep several build for Launchpad PPA on some Major Releases

### DISCUSSION THREAD

**Discussion by chez14** [obsproject/obs-studio#5103] [General]
Hi OBS Team,

I'd like to suggest on keeping some minors and patches build on the PPA, on several major releases. For example, for the `27.x` releases, we keep the 3 newest minor build: `27.0.x`, `27.1.x`, and `27.2.x`. So people have options to use the older one, just in case if the latest release have problem. Here's the sample case of keep the last 3 versions of patch and minor releases in the last 3 major releases.

| Major | Version Kept                                               |
| ----- | ---------------------------------------------------------- |
| 27.x  | `27.0.0`, `27.0.1`,`27.0.3` | `27.1.3`, `27.1.4`,`27.1.5`  |
| 26.x  | `26.0.0`, `26.0.1`, `26.0.2` |`26.1.0`, `26.1.1`, `26.1.2` |
| 25.x  | `25.0.6`, `25.0.7`, `25.0.8`                               |

We can delete older version to keep the PPA Repo on 2GB cap.

To downgrade we can just do:
```sh
sudo apt install obs-studio=27.0.0-0obsproject1
```

To list versions on the PPA we can execute (not sure):
```sh
sudo apt list -a obs-studio
```

### Back Story
I just realized today that [OBS have bug on the drag n' drop feature](https://discord.com/channels/348973006581923840/636682839382949888/874049958834212864) on Scene and Source tab. Upon reaching the support channel on Discord, it is [confirmed](https://discord.com/channels/348973006581923840/636682839382949888/874050312485347379) that the feature is disabled due to a bug that causes data loss.

Ace said that the deb file can be grabbed via Launchpad, but unfortunately the binary files has been deleted =3=.

![image](https://user-images.githubusercontent.com/5555026/128648064-6369c28f-df81-4726-9456-5c9096ab7d67.png)

Experienced Ubuntu/Linux user may just build them from the repo with little-to-no problem, but not all people who use Linux is very fluent with building the binary files. Hence, providing alternatives like downgrading one minor/patch version via PPA is much favorable in my opinion.

### Alternatives
[Another suggestion from Ace](https://discord.com/channels/348973006581923840/636682839382949888/874054484362412042) on Discord is to keep the binary files on the GH Release.

Thank you so much!

**Comment by derrod** [comment]
@Gol-D-Ace can correct me on this but I believe this is not possible? I think PPAs just automatically delete superseeded builds and there's no way to turn that off.

**Reply by Gol-D-Ace** [reply]
Correct I don't know a method that allow them to be stored in the PPA / APT repo.
